src/commonMain/kotlin/au/id/micolous/metrodroid/card/felica/FelicaReader.kt

```diff
@@ -130,6 +141,15 @@ object FelicaReader {
 
                 val services = mutableMapOf<Int, FelicaService>()
 
+                if (onlyFirst && systemNumber > 0) {
+                    // Instead, insert a dummy service with no service codes.
+                    Log.d(TAG, "Not reading system code ${systemCode.hexString}: " +
```

Comment:
It may merit higher level

Reply:
On iOS, this warning would basically always fire; and we don't have this problem on other platforms.  I made this an `INFO` but making this much higher makes iOS logging noisy.